### ISSUE TITLE
Question about Enhancing Interaction within Active Queries

### ISSUE THREAD

**Issue by Dumeowmeow** [GeWu-Lab/bias_in_AVS#2]
Thanks for sharing the code! In the paper, you mentioned that 'audio interactions should only occur between active queries related to perceived latent semantics and only active query features are updated. The dormant query features are masked during backpropagation.' Which section of the code corresponds to this? Is it implemented using the attention mask output by Mask2FormerMaskPredictor? But even if dormant queries are included as learnable queries, I wonder that is it still possible that they are not masked in the attention mask?

**Comment by PeiwenSun2000** [comment]
Thanks for your attention.
Please kindly refer to the transformers package in this project.
i.e. [https://github.com/GeWu-Lab/bias_in_AVS/blob/main/AVSS/transformers/models/mask2former](https://github.com/GeWu-Lab/bias_in_AVS/tree/main/AVSS)
